addons/arsenal/functions/fnc_fillLoadoutsList.sqf

```diff
@@ -104,7 +104,7 @@ if (GVAR(currentLoadoutsTab) != IDC_buttonSharedLoadouts) then {
         } else {
             if (_unavailableItemsList isNotEqualTo []) then {
                 _contentPanelCtrl lnbSetColor [[_newRow, 1], [1, 1, 1, 0.25]]; // Gray
-                if GVAR(unavailableItemsTooltip) then {
```

Comment:
The way it was is valid though, should have the `()` cause that's the standard, but this wouldn't cause any issues

Reply:
The _only_ thing I can think of that could be the issue is same frame execution... But that'd make absolutely zero sense for a tooltip and executing next frame was not resolving the issue - quite possibly user error though given it is me

Reply:
https://community.bistudio.com/wiki/lnbSetTooltip

Not too well versed with GUI but appears so to me